hmiddleware/logger.go

```diff
@@ -0,0 +1,87 @@
+package hmiddleware
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/go-chi/chi"
+	"github.com/go-chi/chi/middleware"
+
+	httprequestid "github.com/heroku/x/requestid"
```

Comment:
I did this because it's what we are doing at https://github.com/heroku/x/blob/839ea2283620b86e226c885c0ac0cb30a3e6014e/grpc/grpchttp/request_id.go#L10

not sure if that is a good reason or not...

Reply:
The reason why it's renamed to `httprequestid ` there is because it's also importing a gRPC requestid package and the names would conflict: https://github.com/heroku/x/blob/839ea2283620b86e226c885c0ac0cb30a3e6014e/grpc/grpchttp/request_id.go#L9-L10

Using the default import name of `requestid` should be fine here. I've also made a note to see if it makes sense to unify (or reorganize) the two requestid packages to prevent the naming conflicts.

Reply:
🆒 

reverted to use default package name at f3418ff